src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -27,6 +27,20 @@ public static PatchMap Generate(WorldGenerationSettings settings, Species defaul
         // Potential starting patches, which must be set by the end of the generating process
         Patch? vents = null;
         Patch? tidepool = null;
+        Patch? banana = null;
+
+        bool useRarePatch = false;
+
+        if (settings.EasterEggs)
+        {
+            // 1% chance for rare biome to spawn (!!CURRENTLY SET TO 1 IN 3 FOR TESTING!!)
+            int rand = random.Next(0, 3);
```

Comment:
I'll put this random call outside the if check to ensure that changing the easter eggs setting doesn't cause the entire patch map to become different with the same seed.